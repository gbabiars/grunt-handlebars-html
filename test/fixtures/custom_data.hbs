<!DOCTYPE html>
<html>
<head lang="en">
    <meta charset="UTF-8">
    <title></title>

    {{#each jsFiles}}<script src="{{this}}"></script>
    {{/each}}
</head>
<body>
{{#if debug}}
    Debug
{{/if}}
</body>
</html>