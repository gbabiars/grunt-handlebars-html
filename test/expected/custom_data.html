<!DOCTYPE html>
<html>
<head lang="en">
    <meta charset="UTF-8">
    <title></title>

    <script src="some_file.js"></script>
    <script src="another_file.js"></script>

</head>
<body>

    Debug

</body>
</html>